cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/ReadSegmentRangeCommand.java

```diff
@@ -114,7 +114,7 @@ private void readAndWriteSegmentToFile(SegmentHelper segmentHelper, String segme
             // TODO: readSegment should take in longs as described here: https://github.com/pravega/pravega/issues/6183
             // TODO: Remove casting to int once 6183 is resolved.
```

Comment:
Please remove the TODO.

Reply:
Yes, that makes sense. Thank you Sandeep. It's removed :)